sdk/cognitiveservices/azure-cognitiveservices-personalizer/azure/cognitiveservices/personalizer/models/__init__.py

```diff
@@ -10,35 +10,106 @@
 # --------------------------------------------------------------------------
 
 try:
-    from .internal_error_py3 import InternalError
-    from .personalizer_error_py3 import PersonalizerError
-    from .error_response_py3 import ErrorResponse, ErrorResponseException
-    from .reward_request_py3 import RewardRequest
-    from .rankable_action_py3 import RankableAction
-    from .rank_request_py3 import RankRequest
-    from .ranked_action_py3 import RankedAction
-    from .rank_response_py3 import RankResponse
+    from ._models_py3 import ApprenticeLearningMetrics
+    from ._models_py3 import ApprenticeModeMetrics
+    from ._models_py3 import DateRange
+    from ._models_py3 import ErrorResponse, ErrorResponseException
+    from ._models_py3 import Evaluation
+    from ._models_py3 import EvaluationContract
+    from ._models_py3 import InternalError
+    from ._models_py3 import LogsProperties
+    from ._models_py3 import LogsPropertiesDateRange
+    from ._models_py3 import ModelProperties
+    from ._models_py3 import MultiSlotRankRequest
+    from ._models_py3 import MultiSlotRankResponse
+    from ._models_py3 import MultiSlotRewardRequest
+    from ._models_py3 import PersonalizerError
+    from ._models_py3 import PolicyContract
+    from ._models_py3 import PolicyReferenceContract
+    from ._models_py3 import PolicyResult
+    from ._models_py3 import PolicyResultSummary
+    from ._models_py3 import PolicyResultTotalSummary
+    from ._models_py3 import RankableAction
+    from ._models_py3 import RankedAction
+    from ._models_py3 import RankRequest
+    from ._models_py3 import RankResponse
+    from ._models_py3 import RewardRequest
+    from ._models_py3 import ServiceConfiguration
+    from ._models_py3 import ServiceStatus
+    from ._models_py3 import SlotRequest
+    from ._models_py3 import SlotResponse
```

Comment:
@lmazuel does this look right? I thought only V3 autorest had .models, but this was generated with `--verison=V2`